azure-pipelines-ci.yml

```diff
@@ -119,6 +119,13 @@ stages:
         Contents: '*.jar'
         TargetFolder: '$(Build.ArtifactStagingDirectory)/agent_deploy'
       condition: and(succeeded(), eq(variables.fullBuild, 'true'))
+    - task: CopyFiles@2
+      displayName: Copy deploy uber files
+      inputs:
+        SourceFolder: 'center/uber_trial/'
```

Comment:
What's the meaning of 'uber_trial'?

Reply:
Just name for the docker's target, or maybe rename it to uber_out_of_box?

Reply:
Could it be just 'uber'?

Reply:
Updated